packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -20,9 +20,11 @@ To better understand SCIM and the specific implementation of SCIM using Okta, se
 
 ### Creating users
 
-The User creation operation brings the user's application profile from Okta over to the Service Provider. A user's application profile represents the key-value pairs defined on the **Profile** tab when a user is added.
+The User creation operation brings the user's application profile from Okta over to the Service Provider. A user's application profile represents the key-value attributes defined on the **Profile** tab when a user is added.
 
-To enable user provisioning, an Okta administrator must configure the provisioning options under the Okta Admin Console. For more information on enabling the provisioning features of your SCIM application, see [Provisioning and Deprovisioning](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Provisioning_Deprovisioning_Overview) under the **Accessing Provisioning Features** section.
+To enable user provisioning, an Okta administrator must configure the provisioning options under the Okta Admin Console. In the Okta Admin Console, select your SCIM application from your list of applications. Under the **Provisioning** tab, click **To App** and **Edit**. In the **Create User** option, click **Enable** and then **Save**.
+
+For more information on enabling the provisioning features of your SCIM application, see [Provisioning and Deprovisioning](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Provisioning_Deprovisioning_Overview) under the **Accessing Provisioning Features** section.
```

Comment:
For a later iteration of this doc, we should try to replace literal links to h.o.c. pages with aliases.

Reply:
Never mind, my mistake.